src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoReadOnly.cs

```diff
@@ -12,17 +12,26 @@ public class DateTimeFormatInfoReadOnly
         public static IEnumerable<object[]> ReadOnly_TestData()
         {
             yield return new object[] { DateTimeFormatInfo.InvariantInfo, true };
+            yield return new object[] { DateTimeFormatInfo.ReadOnly(new DateTimeFormatInfo()), true };
```

Comment:
Ok, i  must  be  seeing  things. I could have sworn when I reviewed this before you were calling ReadOnly on Invariant rather than on a new instance. Nevermind.